### PR TITLE
This commit replaces the deprecated (v.2, Python) Transifex client wi…

### PR DESCRIPTION
**Pull request by why-not-try-calmer**
…th the up-to-date (v.3, Go) official successor.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -15,11 +15,16 @@ jobs:
...
+        env:
+          TX_TOKEN: ${{ secrets.TX_TOKEN }}
         run: |
-          TX_TOKEN=${{ secrets.TX_TOKEN }} make transifex_sync # push
-          TX_TOKEN=${{ secrets.TX_TOKEN }} make transifex_pull
+          curl -OL https://github.com/transifex/cli/releases/download/v1.3.1/tx-linux-amd64.tar.gz
+          tar -xvzf tx-linux-amd64.tar.gz
+          ./tx push
+          ./tx pull
+          ./tx status
```
Comment (m-kuhn) [inline .github/workflows/release.yml]: @why-not-try-calmer is it possible, that the Makefile did more than just push and pull (like extracting translatable strings from docs?)
Maybe it's safer to call `tx` inside the Makefile and preserve those dependencies.

Comment (why-not-try-calmer) [reply .github/workflows/release.yml]: On it.

**Review by m-kuhn — APPROVED** [review]

**Comment by ponceta** [comment]
@why-not-try-calmer is it necessary to have the binary files and the LICENCE file?

Otherwise add these to the .gitignore to avoid having these duplicated in the docs.

**Comment by why-not-try-calmer** [comment]
> @why-not-try-calmer is it necessary to have the binary files and the LICENCE file?
> 
> Otherwise add these to the .gitignore to avoid having these duplicated in the docs.

Sure thing (done!)

**Review by m-kuhn — APPROVED** [review]

**Comment by why-not-try-calmer** [comment]
So apparently the CI does not like the name of the branch. Need to PR again then.

**Pull request closed without merging**